aiida_common_workflows/workflows/relax/gpaw/protocol.yml

```diff
@@ -30,12 +30,9 @@ moderate:
         gpts:
           '@function': 'PW'
           args: {'ecut': 600}
-        convergence:
-          energy: 0.0000000001
-          density: 0.000001
```

Comment:
Did you really mean to delete these lines just to fix the pre-commit? Don't think they cause the pre-commit to fail, did they?

Reply:
Unrelated - this was removed independent of the pre-commit.